tests/v2/validation/projects/projects_additional_test.go

```diff
@@ -0,0 +1,150 @@
+	leaderPodName, err := getRancherLeaderPod(pr.client)
+	require.NoError(pr.T(), err)
+	errorRegex := `\[ERROR\] error syncing '(.*?)': handler mgmt-auth-crtb-controller: .*? namespaces "(.*?)" (?:not found|is forbidden), requeuing`
+	err = checkPodLogsForErrors(pr.client, localCluster, leaderPodName, RancherNamespace, errorRegex, testStartTime, testEndTime)
+	require.NoError(pr.T(), err)
+
+	log.Info("Remove the finalizer that was previously added to the project.")
+	finalizer = nil
+	_, err = updateProjectNamespaceFinalizer(pr.client, createdProject.Name, createdProject.Namespace, finalizer)
+	require.NoError(pr.T(), err, "Failed to update finalizer.")
+
+	log.Info("Verify that the project is deleted.")
+	listOpt := metav1.ListOptions{
+		FieldSelector: "metadata.name=" + createdProject.Name,
+	}
+	project, err := projectsApi.ListProjects(pr.client, createdProject.Namespace, listOpt)
+	require.NoError(pr.T(), err)
+	require.Empty(pr.T(), project, "Project still exists.")
+}
+
+func (pr *ProjectsAdditionalTestSuite) TestUserAdditionToProjectWithTerminatingProjectNamespace() {
+	subSession := pr.session.NewSession()
+	defer subSession.Cleanup()
+
+	log.Info("Create a standard user.")
+	createdUser, err := users.CreateUserWithRole(pr.client, users.UserConfig(), standardUser)
+	require.NoError(pr.T(), err)
+	pr.T().Logf("Created user: %v", createdUser.Username)
+
+	log.Info("Create a project in the downstream cluster.")
+	createdProject, err := createProject(pr.client, pr.cluster.ID)
+	require.NoError(pr.T(), err)
+	err = waitForFinalizerToUpdate(pr.client, createdProject.Name)
+	require.NoError(pr.T(), err)
+
+	log.Info("Simulate a project stuck in terminating state by adding a finalizer to the project.")
+	finalizer := append([]string{dummyFinalizer}, createdProject.Finalizers...)
+	_, err = updateProjectNamespaceFinalizer(pr.client, createdProject.Name, createdProject.Namespace, finalizer)
+	require.NoError(pr.T(), err, "Failed to update finalizer.")
+
+	log.Info("Delete the Project.")
+	err = projectsApi.DeleteProject(pr.client, createdProject)
+	require.NoError(pr.T(), err)
+
+	log.Info("Add the standard user to the project as project owner.")
+	_, err = createProjectRoleTemplateBinding(pr.client, createdUser, createdProject, roleProjectOwner)
+	require.Error(pr.T(), err)
+	pattern := regexp.MustCompile(`projectroletemplatebindings\.management\.cattle\.io ".+?" is forbidden: unable to create new content in namespace ` + regexp.QuoteMeta(createdProject.Name) + ` because it is being terminated`)
+	require.Regexp(pr.T(), pattern, err.Error())
+
+	log.Info("Remove the finalizer that was previously added to the project.")
+	finalizer = nil
+	_, err = updateProjectNamespaceFinalizer(pr.client, createdProject.Name, createdProject.Namespace, finalizer)
+	require.NoError(pr.T(), err, "Failed to update finalizer.")
+
+	log.Info("Verify that the project is deleted.")
```

Comment:
Same as above comment, do we want to have this validation, given it will be taken care by the cleanup logic?